### PR TITLE
app_reviews_by_users

### PR DESCRIPTION
**Pull request by darshan-gandhi**
Software Applications User Reviews

### CODE REVIEW

**Comment by darshan-gandhi** [comment]
Hi @lhoestq  

I have added the readme file as well, please if you could check it once 

Thank you

**Comment by ijaz314** [comment]
Hii please follow me

File: datasets/app_reviews/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+YAML tags:
```
Comment (yjernite) [inline datasets/app_reviews/README.md:2]:
```suggestion
```

**Review by lhoestq — APPROVED** [review]
Awesome thanks :)

**Pull request merged**